pkg/hub/submarinerbrokerinfo/brokerinfo.go

```diff
@@ -214,6 +214,10 @@ func applySubmarinerConfig(brokerInfo *SubmarinerBrokerInfo, submarinerConfig *c
 	setIfValueNotDefault(&brokerInfo.CatalogStartingCSV, submarinerConfig.Spec.SubscriptionConfig.StartingCSV)
 	setIfValueNotDefault(&brokerInfo.InstallPlanApproval, submarinerConfig.Spec.SubscriptionConfig.InstallPlanApproval)
 
+	if brokerInfo.CatalogChannel != defaultCatalogChannel {
+		logger.Warningf("Tracking non-default catalog channel %q (default is %q)", brokerInfo.CatalogChannel, defaultCatalogChannel)
```

Comment:
Why is this a warning? Is this something a user isn't supposed to do?

Reply:
In most cases they shouldn’t, and in all cases support needs to know about it. It could be an info-level message but then it wouldn’t necessarily be logged...

Reply:
If you use `logger.V(...).Infof` it may not be but if you use `logger.Infof`, it should always be logged.